### PR TITLE
Update the Desktop (WPF and UWP) version of the DisplayLayerViewState to be uniform with the other SDK versions.

### PR DESCRIPTION
**Pull request by dkemlage**
Made changes for the DisplayLayerViewState sample to be uniform with  the other SDK versions. Modified both C# and VB.NET versions. Simplified the code to more applicable for a LibRefAPI code sample rather than a platform specific MVVM/data-binding best practices sample.

### CODE REVIEW

**Comment by dkemlage** [comment]
@ThadT Could you please do the code-review on this issue?

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/DisplayLayerViewState/DisplayLayerViewState.xaml [hunk 1]
```diff
@@ -15,19 +15,9 @@
             <StackPanel>
```
Comment (ThadT) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/DisplayLayerViewState/DisplayLayerViewState.xaml]: Make layer information panel a little more compact, center and bold the title.
<Border
            Background="White" BorderBrush="Black" BorderThickness="1"
            HorizontalAlignment="Right" VerticalAlignment="Top"
            Margin="30" Padding="20" Width="**250**">
            <StackPanel>
                <TextBlock Text="Layer Display Status" **FontWeight="Bold" 
                      HorizontalAlignment="Center"**/>
                <Label x:Name="StatusLabel_TiledLayer" Content=""/>
                <Label x:Name="StatusLabel_ImageLayer" Content=""/>
                <Label x:Name="StatusLabel_FeatureLayer" Content=""/>
            </StackPanel>
        </Border>

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/DisplayLayerViewState/DisplayLayerViewStateVB.xaml [hunk 2]
```diff
@@ -16,19 +16,9 @@
             <StackPanel>
```
Comment (ThadT) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/DisplayLayerViewState/DisplayLayerViewStateVB.xaml]: Make layer information panel a little more compact, center and bold the title.
```
<Border
            Background="White" BorderBrush="Black" BorderThickness="1"
            HorizontalAlignment="Right" VerticalAlignment="Top"
            Margin="30" Padding="20" Width="250">
            <StackPanel>
                <TextBlock Text="Layer Display Status" FontWeight="Bold" 
                      HorizontalAlignment="Center"/>
                <Label x:Name="StatusLabel_TiledLayer" Content=""/>
                <Label x:Name="StatusLabel_ImageLayer" Content=""/>
                <Label x:Name="StatusLabel_FeatureLayer" Content=""/>
            </StackPanel>
        </Border>
```

File: src/Windows/ArcGISRuntime.Windows.Samples/Samples/MapView/DisplayLayerViewState/DisplayLayerViewState.xaml [hunk 3]
```diff
@@ -14,16 +14,10 @@
...
-                <DataTemplate>
-                    <TextBlock Margin="2">
-                        <Run Text="{Binding LayerName, Mode=OneWay}"/>
-                        <Run Text=" - " />
-                        <Run Text="{Binding LayerViewStatus, Mode=OneWay}"/>
-                    </TextBlock>
-                </DataTemplate>
-            </ListView.ItemTemplate>
-        </ListView>
+        <StackPanel Orientation="Vertical">
```
Comment (ThadT) [inline src/Windows/ArcGISRuntime.Windows.Samples/Samples/MapView/DisplayLayerViewState/DisplayLayerViewState.xaml]: Add border (with white background) to make the text more readable (especially if it appears over the "for beta use only" watermark).
```
<Border
    Background="White" BorderBrush="Black" BorderThickness="1"
    HorizontalAlignment="Left" VerticalAlignment="Top"
    Margin="30" Padding="20" Width="250">
    <StackPanel Orientation="Vertical">
        <TextBlock x:Name="StatusLabel_TiledLayer" Text=""/>
        <TextBlock x:Name="StatusLabel_ImageLayer" Text=""/>
        <TextBlock x:Name="StatusLabel_FeatureLayer" Text=""/>
    </StackPanel>
</Border>
```

File: src/Windows/ArcGISRuntime.Windows.Samples/Samples/MapView/DisplayLayerViewState/DisplayLayerViewStateVB.xaml [hunk 4]
```diff
@@ -14,16 +14,10 @@
...
-                <DataTemplate>
-                    <TextBlock Margin="2">
-                        <Run Text="{Binding LayerName, Mode=OneWay}"/>
-                        <Run Text=" - " />
-                        <Run Text="{Binding LayerViewStatus, Mode=OneWay}"/>
-                    </TextBlock>
-                </DataTemplate>
-            </ListView.ItemTemplate>
-        </ListView>
+      <StackPanel Orientation="Vertical">
```
Comment (ThadT) [inline src/Windows/ArcGISRuntime.Windows.Samples/Samples/MapView/DisplayLayerViewState/DisplayLayerViewStateVB.xaml]: Add border (with white background) to make the text more readable (especially if it appears over the "for beta use only" watermark).
```
<Border
    Background="White" BorderBrush="Black" BorderThickness="1"
    HorizontalAlignment="Left" VerticalAlignment="Top"
    Margin="30" Padding="20" Width="250">
    <StackPanel Orientation="Vertical">
        <TextBlock x:Name="StatusLabel_TiledLayer" Text=""/>
        <TextBlock x:Name="StatusLabel_ImageLayer" Text=""/>
        <TextBlock x:Name="StatusLabel_FeatureLayer" Text=""/>
    </StackPanel>
</Border>
```

**Comment by ThadT** [comment]
@dkemlage A couple minor UI things to make the layer text a bit more readable. Ready to merge.

**Comment by dkemlage** [comment]
@ThadT Thanks for the QA on the WPF and UWP versions of the DisplayLayerViewState sample. I made both the UI changes and fixed the typos that you found in the Xamarin versions.

**Pull request merged**